internal/console/kubevirt_backend.go

```
@@ -161,6 +163,47 @@ func (b *kubeVirtBackend) Connect(ctx context.Context, target Target) (io.ReadWr
+	if err != nil {
+		return nil, fmt.Errorf("failed to build auth wrappers: %w", err)
+	}
+
+	req, err := http.NewRequest(http.MethodGet, "https://placeholder", nil)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create request: %w", err)
+	}
+
+	// RoundTrip populates the request with auth headers, then our capture
+	// round-tripper saves them and returns a sentinel error.
+	_, _ = rt.RoundTrip(req)
```

golangci-lint (2.11.4)

[error] response body must be closed

(bodyclose)